### PR TITLE
Add a dependencies section in the GitHub task template

### LINKED ISSUE #17: Add a Dependencies section in the new issue task template
### Aim

Add a new `Dependencies` block in the [`Task`](https://github.com/SofairOfficial/rust-repository-template/blob/main/.github/ISSUE_TEMPLATE/task.yml) GithHub template.

### Assumptions

- None

### Acceptance criteria

- [x] Add the `Dependencies` block between `Risks` and `References` blocks in the `Task` template (see image below)
image

- [x] The description for the new `Dependencies` block is "Indicate any dependencies that could block the execution of this task"

- [x] Remove the `[TASK]` prefix in the task title (see image below)
image

### Risks

- None

### References

- None

### PR DESCRIPTION
**Pull request by GuiBL4**
## Description

The `Dependencies` section has been added in the task template.
The title field of task template is deleted (unecessary)

## Related issues

This PR closes #17 

## How has this been tested?

- Not relevant

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by binadamu-isiyoonekana — APPROVED** [review]
Great!

File: .github/ISSUE_TEMPLATE/task.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 name: Task
 description: Submit new Task
-title: "[TASK]: "
```
Comment (binadamu-isiyoonekana) [inline .github/ISSUE_TEMPLATE/task.yml:3]: The title is in fact not a mandatory field.

**Review by binadamu-isiyoonekana — CHANGES_REQUESTED** [review]
Testbed

**Review by hmapedro — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/task.yml (hunk 1 above)
Comment (GuiBL4) [reply .github/ISSUE_TEMPLATE/task.yml:3]: Yes it can be deleted.

**Review by binadamu-isiyoonekana — APPROVED** [review]
Perfect.

**Pull request merged**